tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/NServiceBus/NServiceBusDriver.cs

```diff
@@ -112,35 +104,33 @@ public void StopNServiceBus()
         [LibraryMethod]
         [Transaction]
         [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
-        public void PublishEventInTransaction()
+        public async void PublishEventInTransaction()
```

Comment:
`async void` is [generally considered bad](https://learn.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void). 

I think these can all safely be changed to `async Task` instead.

Reply:
Good catch, done.